swagger/swagger.yaml

```diff
@@ -98,11 +98,12 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
-  /scans/{scanId}/launch:
+  '/scans/{scanId}/launch':
     post:
       summary: Launch a scan
-      description: |
+      description: >
```

Comment:
I just see that there is a difference between `> Folded Style` and `| literal style`, see http://stackoverflow.com/a/21699210. Don't know which one is the best (maybe we should take care of the Swagger-UI result) but I think we should use only one form in the whole document. What's your thought?